diff --git a/build.gradle.kts b/build.gradle.kts
index d4f97eb6..a2f0978f 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -104,6 +104,7 @@ dependencies {
         implementation(libs.micronaut.json.schema.generator)
     }
     implementation(project(":libsignal-cli"))
+    implementation(files("/tmp/libsignal-client.jar"))
 
     testImplementation(libs.junit.jupiter)
     testImplementation(platform(libs.junit.jupiter.bom))
@@ -120,6 +121,10 @@ configurations {
     }
 }
 
+configurations.all {
+    exclude(group = "org.signal", module = "libsignal-client")
+}
+
 
 tasks.withType<AbstractArchiveTask>().configureEach {
     isPreserveFileTimestamps = false
